utils/numericFormatters.js

```diff
@@ -1,3 +1,14 @@
 export function addComma(value) {
   return value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
 }
+
+export function formatPhoneNum(value, parensStyle = false) {
```

Comment:
What if, to make this more scalable, we make this second param a `config` object? Then `parensStyle` could be a possible property of that object, plus we could add more config options in the future (e.g.: international formats, country code, dashes, etc).
```
formatPhoneNum(value, { parensStyle = false } = {}) {
```

Reply:
dun dun dun dooooone :)